src/chromium/crPage.ts

```diff
@@ -67,7 +68,7 @@ export class CRPage implements PageDelegate {
     this._mainFrameSession = new FrameSession(this, client, targetId);
     this._sessions.set(targetId, this._mainFrameSession);
     client.once(CRSessionEvents.Disconnected, () => this._page._didDisconnect());
-    this._pagePromise = this._mainFrameSession._initialize().then(() => this._initializedPage = this._page).catch(e => e);
+    this._pagePromise = this._mainFrameSession._initialize(!!isHeadful).then(() => this._initializedPage = this._page).catch(e => e);
```

Comment:
nit: isHeadful is always defined, make it non-optional

Reply:
Done